opencti-platform/opencti-front/src/private/components/settings/Policies.tsx

```diff
@@ -213,7 +214,7 @@ const PoliciesComponent: FunctionComponent<PoliciesComponentProps> = ({
                           <EETooltip>
                             <ObjectOrganizationField
                               name="platform_organization"
-                              disabled={disabled || !isEnterpriseEdition || (isSensitiveModificationEnabled && !isAllowed)}
+                              disabled={disabled || !isEnterpriseEdition || (isSensitive && !isAllowed)}
```

Comment:
Since the DangerZoneBlock already provides the logic of disabling, you can simplify it to                               disabled={disabled || !isEnterpriseEdition}

Reply:
done